docs/guides/monitor/statsd.md

```diff
@@ -0,0 +1,297 @@
+- If we have multiple instances of the same metric, we can define different contexts, (Optional).
+- We define a dimension according to the syntax we highlighted above.
+- We define a type for each synthetic chart (line, area, stacked)
+- We define the units for each synthetic chart. 
+
+Following the above steps, we append to the `k6.conf` that we defined above, the following configuration:
+
+```
+[http_req_total]
+    name = http_req_total
+    title = Total HTTP Requests
+    family = http requests
+    context = k6.http_requests
+    dimension = k6.http_reqs http_reqs last 1 1 sum
+    type = line
+    units = requests/s
+	
+[vus]
+    name = vus
+    title = Virtual Active Users
+    family = k6_metrics
+    dimension = k6.vus vus last 1 1 
+    dimension = k6.vus_max vus_max last 1 1 
+    type = line
+    unit = vus
+	
+[iteration_duration]
+    name = iteration_duration_2
+    title = Iteration duration
+    family = k6_metrics
+    dimension = k6.iteration_duration iteration_duration last 1 1 
+    dimension = k6.iteration_duration iteration_duration_max max 1 1 
+    dimension = k6.iteration_duration iteration_duration_min min 1 1 
+    dimension = k6.iteration_duration iteration_duration_avg avg 1 1 
+    type = line
+    unit = s
+
+[dropped_iterations]
+    name = dropped_iterations
+    title = Dropped Iterations
+    family = k6_metrics
+    dimension = k6.dropped_iterations dropped_iterations last 1 1 
+    units = iterations
+    type = line
+
+[data]
+    name = data
+    title = K6 Data
+    family = k6_metrics
+    dimension = k6.data_received data_received last 1 1
+    dimension = k6.data_sent data_sent last -1 1
+    units = kb/s
+    type = area 
+
+[http_req_status]
+    name = http_req_status
+    title = HTTP Requests Status
+    family = http requests
+    dimension = k6.http_req_blocked http_req_blocked last 1 1 
+    dimension = k6.http_req_connecting http_req_connecting last 1 1
+    units = ms
+    type = line
+    
+[http_req_duration]
+    name = http_req_duration
+    title = HTTP requests duration
+    family = http requests
+    dimension = k6.http_req_sending http_req_sending last 1 1
+    dimension = k6.http_req_waiting http_req_waiting last 1 1
+    dimension = k6.http_req_receiving http_req_receiving last 1 1
+    units = ms
+    type = stacked
+```
+
+> Take note that Netdata will report the rate for metrics and counters, even if k6 or another application sends an _absolute_ number. For example, k6 sends absolute HTTP requests with `http_reqs`, but Netdat visualizes that in `requests/second`.
+
+To enable this StatsD configuration, [restart Netdata](/docs/configure/start-stop-restart.md).
+
+## Final touches
+
+At this point, you have used StatsD to gather metrics for k6, creating a whole new section in your Netdata dashboard in the process. Uil can further customize the icon of the particular section, as well as the description for each chart.
```

Comment:
What's `Uil`?

Reply:
nice catch @jsoref !

Reply:
Uil is actually an [institute](https://uil.unesco.org/unesco-institute)!

But thanks for the catch @jsoref 😅

Reply:
Fwiw, that's from https://github.com/check-spelling/netdata/commit/cd752b3b3057f5f54171cd7525737e1a925b32f3#commitcomment-49502749

I'll send a PR for everything else. But this one I can't figure out at all.

If you're up for it, I think netdata would benefit from running the check-spelling action 🙂 .

Reply:
> But this one I can't figure out at all .

Yeah, we know @joelhans  :P . Thanks, in any case, we will take it from here.

As fas the GHA, I think @joelhans  @Ferroin have chatted about this. It's seems nice to have this for good measure.

Reply:
Ah, so it should be `UIL` per their page:
> UNESCO Institute for Lifelong Learning (UIL)

I can toss that in w/ everything else :-).